### PR TITLE
Review stories sf

### PR DESCRIPTION
**Pull request by StephanieFriend**
- Changed items index route in merchant namespace from /items to /merchant_id/items
- Adjusted the merchant items controller index action to include the merchant_id param
- Removed the most and least popular logic from the partial and placed it only in the main item view
- Add a conditional to prevent a merchant employee from editing other merchant's items

### CODE REVIEW

**Comment by StephanieFriend** [comment]
-Updated tests to reflect route changes
- Change inventory conditional from greater_than to greater_than_or_equal_to

**Comment by StephanieFriend** [comment]
Updated with Daniels review branch

File: app/controllers/merchant/items_controller.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Merchant::ItemsController < Merchant::BaseController
 
   def index
-    @merchant = Merchant.find(current_user.merchant_id)
+    @merchant = Merchant.find(params[:merchant_id])
```
Comment (margoflewelling) [inline app/controllers/merchant/items_controller.rb:4]: Good idea.

File: app/controllers/merchant/items_controller.rb [hunk 2]
```diff
@@ -35,14 +35,14 @@ def status
       @item.update(active?: true)
       flash[:notice] = "#{@item.name} is now available for sale"
     end
-    redirect_to "/merchant/items"
+    redirect_to "/merchant/#{@item.merchant.id}/items"
```
Comment (margoflewelling) [inline app/controllers/merchant/items_controller.rb:38]: Nice catch.

File: app/models/item.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@ class Item <ApplicationRecord
                         :image,
                         :inventory
   validates_inclusion_of :active?, :in => [true, false]
-  validates_numericality_of :inventory, :price, greater_than: 0
+  validates_numericality_of :inventory, :price, greater_than_or_equal_to: 0
```
Comment (margoflewelling) [inline app/models/item.rb:13]: Yes this makes more sense.

**Pull request merged**